osism/commands/manage.py

```diff
@@ -225,6 +225,12 @@ def get_parser(self, prog_name):
             help="Hide images that should be deleted",
             action="store_true",
```

Comment:
not directly related, but this argument has `store_true` and default=True, which sounds useless to me. more importantly, this should be exclusive in regard to the `--delete` option? Doesn't make sense to both hide and delete images at the same time?